packages/core/src/utils/ConstantsUtil.ts

```diff
@@ -5,10 +5,23 @@ export const ONRAMP_PROVIDERS = [
     label: 'Coinbase',
     name: 'coinbase',
     feeRange: '1-2%',
-    url: ''
+    url: '',
+    supportedChains: ['evm']
+  },
+  {
+    label: 'Meld.io',
+    name: 'meld',
+    feeRange: '1-2%',
+    url: 'https://meldcrypto.com',
+    supportedChains: ['evm', 'solana']
   }
 ]
 
+export const MELD_DEV_PUBLIC_KEY =
+  '?publicKey=WXETMsajb7XcQBm7mcxAab:q3MtzJpiEMtXVNXsqYkAnAaBkgStybGVtZ'
```

Comment:
Addition to using `URL` class comment, keeping query param with `?publicKey=` prefix is not a good approach, instead we should generate query params on top, this variable should only keep the key.